frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form.tsx

```diff
@@ -180,9 +183,16 @@ const AdvancedSection: React.FC<AdvancedSectionProps> = ({
                   });
                 }}
                 provisioner={provisioner}
-                accessMode={state.accessMode?.value}
+                accessMode={
+                  isSPSettingProvided && applySP
```

Comment:
Same issue as above, I think this condition should already be known to be true.

Reply:
changed to `state.accessMode?.value || spAccessMode?.getValue()` to get initial value from SP when `state.accessMode?.value` is `undefined`